src/renderer/components/FtCommunityPost/FtCommunityPost.vue

```diff
@@ -50,6 +50,17 @@
       >
         {{ publishedText }}
       </p>
+      <div class="header-share-button">
+        <FtShareButton
+          v-if="postId"
+          :id="postId"
+          share-target-type="Post"
+          dropdown-position-y="bottom"
+          theme="base-no-default"
+          :use-shadow="false"
+          :size="16"
```

Comment:
Please revert these theming changes so that the share button looks the same as it does everywhere else.